theano/sandbox/cuda/tests/test_extra_ops.py

```diff
@@ -162,7 +162,7 @@ def test_GpuCumsum2D(self):
             utt.assert_allclose(np.cumsum(a, axis=axis), f(a))
 
             # Use multiple GPU gridblocks
-            a_shape = [5, 5]
+            a_shape = [4, 4]
```

Comment:
why lower the shape size? To lower numerical problem in the test? That is my guess, so I think it is ok.